Sources/MessagingPush/MessagingPushConfigOptions.swift

```diff
@@ -1,21 +1,33 @@
-public struct MessagingPushConfigOptions {
-    // Used to create new instance of MessagingPushConfigOptions when the MessagingPushAPN module is configured.
-    // Each property of the MessagingPushConfigOptions object can be modified by the user.
-    public enum Factory {
-        public static func create() -> MessagingPushConfigOptions {
-            MessagingPushConfigOptions(
-                autoFetchDeviceToken: true
-            )
-        }
-    }
+import CioInternalCommon
 
-    public enum Keys: String { // Constants used to map each of the options in MessagingPushConfigOptions
-        case autoFetchDeviceToken
+public struct MessagingPushConfigOptions {
+    public init() {
+        self.autoFetchDeviceToken = true
+        self.showPushAppInForeground = true
     }
 
     /**
      Enable automatic fetching of device token by the SDK without the need to write custom code by the customer.
      On fetching the token, SDK will auto-register the device. This value is `true` by default.
      */
     public var autoFetchDeviceToken: Bool
+
+    /**
+     Display push notifications sent by Customer.io while app is in foreground. This value is `true` by default.
+     */
+    public var showPushAppInForeground: Bool
```

Comment:
Added this SDK configuration to the messaging module config instead of SdkConfig since we are planning on encapsulating config options into modules.